spec/controllers/users_spec.rb

```diff
@@ -0,0 +1,62 @@
+ require_relative '../spec_helper'
+
+describe UsersController do
+  context 'with authenticated user logged in' do
```

Comment:
This isn't right, when user is about to sign up to your app - it should not be expected for him to be authenticated. He does not have the account yet, how can you authenticate him?